app/controllers/games_controller.rb

```diff
@@ -29,35 +27,18 @@ def play_game
   end
 
   def show
-    resume_game = Game.where(:id => params[:id]).first
+    resume_game = Game.find_by(:id => params[:id])
     if !resume_game.nil?
-      player = resume_game.player
-      # @opponent = resume_game.opponent
-      current_player = resume_game.current_player
-      board = resume_game.board
-      toggle = TicTacToeGame::Toggle.new(player)
-      game = TicTacToeGame::Game.new(board, player, toggle)
-      # winner =  get_toggle.current_turn(current_player)
-      game_over = GameoverType.new
-      answer = game_over.message(game, current_player)
-      render json: { data:{message: answer,id: resume_game.id} }, status: 200
+      answer = ShowAssignValue.new.set_value(resume_game)
```

Comment:
See if you can come up with better names for: `answer`, `ShowAssignValue`, and `set_value`.

I usually think about what higher purpose does the class/method serve and not how it is implemented. Focus on the `why` rather than the `how`.